src/layout.cc

```diff
@@ -1145,13 +1145,17 @@ bool ParseConditionTable(const ots::Font *font,
     return OTS_FAILURE_MSG("Axis index out of range in condition");
   }
 
-  // Check min/max values are within range -1.0 .. 1.0 and properly ordered
+  // Check min/max values are within range -1.0 .. 1.0.
   if (filter_range_min_value < -0x4000 || // -1.0 in F2DOT14 format
-      filter_range_max_value > 0x4000 || // +1.0 in F2DOT14 format
-      filter_range_min_value > filter_range_max_value) {
+      filter_range_max_value > 0x4000) // +1.0 in F2DOT14 format
```

Comment:
```suggestion
      filter_range_max_value > 0x4000) { // +1.0 in F2DOT14 format
```

Reply:
yeah, thanks - I don't currently have a local build set up so this was untested :(